cmd/segment-reaper/observer.go

```diff
@@ -34,10 +36,12 @@ type object struct {
 // name.
 type bucketsObjects map[string]map[storj.Path]*object
 
-func newObserver(db metainfo.PointerDB, w *csv.Writer) *observer {
+func newObserver(db metainfo.PointerDB, w *csv.Writer, from, to *time.Time) *observer {
 	return &observer{
 		db:     db,
 		writer: w,
```

Comment:
What do you think about making this constructor responsible for writing the headers (instead of in detect.go:71-81)?

Reply:
I would prefer to have a constructor without returning error but on the other hand make more sense to write headers in observer. I will move code.

Reply:
done